.github/workflows/scheduler.yml

```diff
@@ -0,0 +1,38 @@
+on: push
+  # schedule:
+  #   # At 05:00 PM UTC
+  #   - cron: "0 17 * * *"
```

Comment:
should this be on: schedule?

Reply:
Yeah, but first do we want to test run this workflow? This repo (and the others) will need a repo scoped access token

Reply:
I added an access token to this repo.  Do we want to wait for a new commit to featuretools to test all the way through?

Already tested the "not recent" scenario

Reply:
For testing, I can just change the recent value to make it run through on the next commit

Reply:
It ran successfully ([run](https://github.com/alteryx/featuretools-tsfresh-primitives/actions/runs/730344416)), I'll revert the changes to schedule